testing/src/main/java/org/oppia/testing/FakeSystemClock.kt

```diff
@@ -4,13 +4,15 @@ import android.os.SystemClock
 import org.robolectric.Robolectric
 import java.util.concurrent.atomic.AtomicLong
 import javax.inject.Inject
+import javax.inject.Singleton
 
 // TODO(#89): Actually finish this implementation so that it properly works across Robolectric and
 //  Espresso, and add tests for it.
 /**
  * A Robolectric-specific fake for the system clock that can be used to manipulate time in a
  * consistent way.
  */
+@Singleton
```

Comment:
Why do we need it to be Singleton?

Reply:
Because it's storing state (an ``AtomicLong``) that we want to be shared among all places that inject this type.

Reply:
Okay got it